Policies/Kubernetes/selinux/rego/src_test.rego

```diff
@@ -0,0 +1,203 @@
+    results := violation with input as input
+    count(results) == 1
+}
+
+test_input_seLinux_options_empty {
+    input := { "review": input_review, "parameters": input_parameters_empty}
+    results := violation with input as input
+    count(results) == 1
+}
+
+test_input_seLinux_options_no_security_context {
+    input := { "review": input_review_no_security_context, "parameters": input_parameters_in_list}
+    results := violation with input as input
+    count(results) == 0
+}
+
+test_input_seLinux_options_two_allowed_in_list {
+    input := { "review": input_review_two, "parameters": input_parameters_in_list}
+    results := violation with input as input
+    count(results) == 0
+}
+
+test_input_seLinux_options_two_subset_allowed_in_list {
+    input := { "review": input_review_two_subset, "parameters": input_parameters_in_list}
+    results := violation with input as input
+    count(results) == 0
+}
+
+test_input_seLinux_options_two_subset_not_allowed_not_in_list {
+    input := { "review": input_review_two_subset, "parameters": input_parameters_not_in_list}
+    results := violation with input as input
+    count(results) == 1
+}
+
+test_input_seLinux_option_two_not_allowed_not_in_list {
+    input := { "review": input_review_two, "parameters": input_parameters_not_in_list}
+    results := violation with input as input
+    count(results) == 1
+}
+
+test_input_seLinux_options_many_allowed_in_list {
+    input := { "review": input_review_many, "parameters": input_parameters_in_list}
+    results := violation with input as input
+    count(results) == 0
+}
+
+test_input_seLinux_options_many_not_allowed_not_in_list {
+    input := { "review": input_review_many, "parameters": input_parameters_not_in_list}
+    results := violation with input as input
+    count(results) == 1
+}
+
+test_input_seLinux_options_many_not_allowed_not_in_list_two {
+    input := { "review": input_review_many, "parameters": input_parameters_not_in_list_two}
+    results := violation with input as input
+    count(results) == 1
+}
+
+input_review = {
+    "object": {
+        "metadata": {
+            "name": "nginx"
+        },
+        "spec": {
+            "containers": input_containers_one,
+      }
+    }
+}
+
+input_review_two = {
+    "object": {
+        "metadata": {
+            "name": "nginx"
+        },
+        "spec": {
+            "securityContext": input_seLinuxOptions
+        }
+    }
+}
+
+input_review_two_subset = {
+    "object": {
+        "metadata": {
+            "name": "nginx"
+        },
+        "spec": {
+            "securityContext": input_seLinuxOptions_subset
+        }
+    }
+}
+
+input_review_no_security_context = {
+    "object": {
+        "metadata": {
+            "name": "nginx"
+        },
+        "spec": {}
+    }
+}
+
+input_review_many = {
+    "object": {
+        "metadata": {
+            "name": "nginx"
+        },
+        "spec": {
+            "containers": input_containers_many,
+      }
+    }
+}
+
```

Comment:
As securityContext can be defined at both Pod and Container level, can you add some tests for this scenario?
https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container